components/exploits/ExploitStatus.tsx

```diff
@@ -9,7 +9,7 @@ interface Props {
   address: string;
   chainId: number;
   exploit: Exploit;
-  status: 'affected' | 'safe';
+  status: 'affected' | 'safe' | 'previously_accessible';
```

Comment:
Let's rename this to `'previously_affected'` to be consistent with the other names.